pkgs/development/python-modules/locationsharinglib/default.nix

```diff
@@ -1,65 +1,56 @@
 {
   lib,
-  betamax,
   buildPythonPackage,
   cachetools,
   coloredlogs,
-  emoji,
   fetchPypi,
-  nose,
   pythonOlder,
   pytz,
   requests,
+  setuptools,
 }:
 
 buildPythonPackage rec {
   pname = "locationsharinglib";
   version = "5.0.3";
-  format = "setuptools";
+  pyproject = true;
 
-  disabled = pythonOlder "3.7";
+  disabled = pythonOlder "3.9";
 
   src = fetchPypi {
     inherit pname version;
     hash = "sha256-ar5/gyDnby0aceqqHe8lTQaHafOub+IPKglmct4xEGM=";
   };
 
-  propagatedBuildInputs = [
-    coloredlogs
-    requests
-    cachetools
-    pytz
-  ];
-
-  nativeCheckInputs = [
-    betamax
-    emoji
-    nose
-  ];
-
   postPatch = ''
     # Tests requirements want to pull in multiple modules which we don't need
     substituteInPlace setup.py \
-      --replace "tests_require=test_requirements" "tests_require=[]"
+      --replace-fail "tests_require=test_requirements" "tests_require=[]"
     substituteInPlace requirements.txt \
-      --replace "coloredlogs>=15.0.1" "coloredlogs" \
-      --replace "pytz>=2023.3" "pytz"
+      --replace-fail "coloredlogs>=15.0.1" "coloredlogs" \
+      --replace-fail "pytz>=2023.3.post1" "pytz"
```

Comment:
Just checking if `pythonRelaxDeps`/`pythonRemoveDeps` might work here?

Reply:
Just tested, it seems to work without this patch, so I've removed it.